### ISSUE TITLE
Extended support for existing `mx.nd` ops in `mx.np(x)`.

### ISSUE THREAD

**Issue by fhieber** [apache/incubator-mxnet#20373]
## Description
We would like to see operators that currently exist in the mx.nd/mx.sym namespace for MXNet v1.x to also be ported to the new np/npx namespaces (MXNet 2.0). The following operators are required for migrating [Sockeye](https://github.com/awslabs/sockeye) to MXNet 2.0.

## Missing operators:
```
mx.nd.contrib.interleaved_matmul_*
mx.nd.contrib.intgemm_*
mx.nd.add_n
mx.nd.SequenceMask
mx.nd.slice_like
```

**Comment by barry-jin** [comment]
Hi @fhieber 
For operators `mx.nd.contrib.interleaved_matmul_*`, `mx.nd.add_n` and `mx.nd.slice_like`, they are ported to npx namespace in PR #20375 as well as some test cases. 
For `mx.nd.contrib.intgemm_*`, they have been ported to npx name space and you can find related test cases [here](https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_contrib_intgemm.py). 
For `mx.nd.SequenceMask`, it is already ported and you can find some implementations in [rnn_cell.py](https://github.com/apache/incubator-mxnet/blob/da4ff3a4dc0bd6a54af3d75c492021d18ba1867b/python/mxnet/gluon/rnn/rnn_cell.py#L86). 
The replacement of `mx.nd.identity` in numpy mode is [`mx.np.copy`](https://github.com/apache/incubator-mxnet/blob/da4ff3a4dc0bd6a54af3d75c492021d18ba1867b/python/mxnet/numpy/multiarray.py#L1594)

**Comment by fhieber** [comment]
Thanks @barry-jin, looking forward to #20375 be merged.
What about `mx.nd.random.multinomial()`? Is there an equivalent in the np(x) namespace?

**Comment by barry-jin** [comment]
Yes, the equivalent is `mx.np.random.multinomial` and can be found [here](https://github.com/apache/incubator-mxnet/blob/7b4d61d11d1ee29b6000588f03fc074c13d2043e/python/mxnet/numpy/random.py#L386-L425).

**Comment by fhieber** [comment]
Hi @barry-jin,

I think `mx.np.random.multinomial` behaves differently. I now use `npx.random.categorical`, which does what we need (drawing 1 sample from multiple distributions (by row)).

I saw that #20375 has been merged. When do these changes show up in nightly builds at `https://dist.mxnet.io/python`? `mxnet-2.0.0b20210627` still doesn't seem to contain your changes (addition of slice_like).
